torch/tensor.py

```diff
@@ -414,17 +414,19 @@ def __rfloordiv__(self, other):
     __ge__ = _C._TensorBase.ge
     __abs__ = _C._TensorBase.abs
 
-    def _std_mean(self, dim=None, unbiased=True, keepdim=False):
+    def std_mean(self, dim=None, unbiased=True, keepdim=False):
```

Comment:
I think this needs two changes:
1) this shouldn't be done in python, it should be done in native_functinos.
2) the method should be called `_std_mean`, because it's just there for testing purposes.

Reply:
@gchanan Can we just make it a Tensor method, if we want to move them to native_functions?

Reply:
it shouldn't just be a method.  I'm suggesting two things:

1) A function called std_mean
2) A method called _std_mean, that is just there for testing purposes.

Reply:
What will be the benefit of doing this in native_functions?

Reply:
the parsing will be consistent and you can maintain everything in one place.

Reply:
I changed autograd test, to be able to avoid fail when there is a method that implemented only in torch